tests/Aspire.Hosting.Azure.Tests/AzureAppConfigurationExtensionsTests.cs

```diff
@@ -105,4 +105,25 @@ public void AddAsExistingResource_ShouldBeIdempotent_ForAzureAppConfigurationRes
         // Assert - Both calls should return the same resource instance, not duplicates
         Assert.Same(firstResult, secondResult);
     }
+
+    [Fact]
+    public async Task AddAsExistingResource_RespectsExistingAzureResourceAnnotation_ForAzureAppConfigurationResource()
+    {
+        using var builder = TestDistributedApplicationBuilder.Create();
+        var existingName = builder.AddParameter("existing-appconfig-name");
+        var existingResourceGroup = builder.AddParameter("existing-appconfig-rg");
+
+        var appConfig = builder.AddAzureAppConfiguration("test-app-config")
+            .AsExisting(existingName, existingResourceGroup);
+
+        var module = builder.AddAzureInfrastructure("mymodule", infra =>
+        {
+            _ = appConfig.Resource.AddAsExistingResource(infra);
+        });
+
+        var (manifest, bicep) = await AzureManifestUtils.GetManifestWithBicep(module.Resource, skipPreparer: true);
+
+        await Verify(manifest.ToString(), "json")
```

Comment:
(super nit) Do we need to baseline the manifest json in these tests? Doesn't seem that valuable IMO.